pyproject.toml

```diff
@@ -0,0 +1,65 @@
+[build-system]
+requires = ["setuptools", "wheel", "versioneer"]
```

Comment:
I had to add `versioneer` here. I'd rather skip it, but found out that w/o it the wheel doesn't build.

Reply:
Ah did not see that comment before my review, sorry!